googlespread.rb

```diff
@@ -0,0 +1,50 @@
+require "google_drive"
+require 'soda/client'
+require "dotenv"
+Dotenv.load
+client = SODA::Client.new({domain: "data.cityofnewyork.us", app_token: "w4imPv49yhvBlkOUukeVAUZeD"})
+response = client.get("feuq-due4")
+puts response[0].values.inspect
+
+# Creates a session. This will prompt the credential via command line for the
+# first time and save it to config.json file for later usages.
+session = GoogleDrive::Session.from_config("config.json")
+
+# First worksheet of
+# https://docs.google.com/spreadsheet/ccc?key=pz7XtlQC-PYx-jrVMJErTcg
+# Or https://docs.google.com/a/someone.com/spreadsheets/d/pz7XtlQC-PYx-jrVMJErTcg/edit?usp=drive_web
+ws = session.spreadsheet_by_key(ENV["GOOGLE_INIT"]).worksheets[0]
+
+# Gets content of A2 cell.
+p ws[1, 1]  = "NY OPENDATA LIBRARIES"#==> "hoge"
+
+# Changes content of cells.
+# Changes are not sent to the server until you call ws.save().
+(1..response.first.keys.length).each do |param|
+  ws[2,param+1] = response.first.keys[param-1]
+end
+
+(0..response.length-1).each do |i|
+    array = response[i].values
+    (0..array.length).each do |data|
+    ws[i+3, data+2] = array[data]
+end
+end
+
```

Comment:
Very close! Looks like there are two small things. One you need a `Gemfile` and the `Gemfile.lock`. 

There is also a second problem. The coordinates are passed back as a hash instead of the actual coordinates.

![image](https://cloud.githubusercontent.com/assets/111761/24986573/030747c8-1fc9-11e7-813f-d2a36bca35f3.png)